modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -117,11 +115,47 @@ spec:
     storage:
       files:
       - contents:
-          source: data:text/plain;base64,e30K
           source: data:text/plain;base64,ewogInVybCI6ICJodHRwczovL3RhbmcuZXhhbXBsZS5jb20iLAogInRocCI6ICJaUk1leTFjR3cwN3psVExHYlhuUWFoUzBHdTAiCn0K
         mode: 420
         overwrite: true
         path: /etc/clevis.json
 EOF
 ----
+
+. Create an additional machineconfig to make sure the node wait for network before trying to reach the Tang server:
+** For worker nodes, use the following command:
```

Comment:
In most cases, do users need to do this for both machine types? If so, the substeps should start with `..` instead of `**`.